can_bus/quadruna/VC/VC_enum.json

```diff
@@ -10,5 +10,16 @@
     "INSUFFICIENT_READINGS": 1, 
     "INTERNAL_ERROR": 2, 
     "LIMIT_EXCEEDED": 3
+  },
+
+  "Efuse_Enum": {
```

Comment:
`EfuseEnum`

Reply:
Also, maybe make another value called `NO_FAULT` or something like that. JSONCAN will transmit the zeroth value by default for everything, so at startup it's going to broadcast `SHDN` for `Efuse` and 0 for `Fault_Count` which is maybe confusing. Maybe manually set the former signal to `NO_FAULT` at startup to avoid this confusion.